index.bs

```diff
@@ -3254,8 +3259,11 @@ credential.
         with more than one [=user account=] at the [=[RP]=].
 
     :   <dfn>displayName</dfn>
-    ::  A [=human palatability|human-palatable=] name for the [=user account=], intended only for display. For example, "Alex Müller" or "田中倫". The
-        [=[RP]=] SHOULD let the user choose this, and SHOULD NOT restrict the choice more than necessary.
+    ::  A [=human palatability|human-palatable=] name for the [=user account=], intended only for
+        display. The [=[RP]=] SHOULD let the user choose this, and SHOULD NOT restrict the choice
+        more than necessary. The [=[RP]=] MAY set this to the same value as {{PublicKeyCredentialEntity/name}}.
```

Comment:
"The [=[RP]=] MAY set this to the same value as {{PublicKeyCredentialEntity/name}}."

I think we should suggest setting the empty string when no suitable value is available, rather than duplicating.

Reply:
I've updated the wording here to suggest it's okay to set this to an empty string if needed.

Reply:
@agl Can you please take a look at this latest change and see if it satisfies your concern?